Animations/src/core/GladLayer.cpp

```diff
@@ -17,11 +17,6 @@ namespace MathAnim
 			g_logger_info("GLAD initialized.");
 			g_logger_info("Hello OpenGL %d.%d", GLVersion.major, GLVersion.minor);
 
-#ifdef _DEBUG
-			glEnable(GL_DEBUG_OUTPUT);
```

Comment:
I'll be adding some fallback support for older OpenGL versions in the future, but for now I'd like to leave these calls in. I've created an issue here to add to the list https://github.com/ambrosiogabe/MathAnimation/issues/60